src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Extensions/RoslynTextChangeExtensions.cs

```diff
@@ -1,15 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT license. See License.txt in the project root for license information.
-
-using Microsoft.VisualStudio.LanguageServer.ContainedLanguage;
-using Microsoft.VisualStudio.Text;
-using RoslynTextChange = Microsoft.CodeAnalysis.Text.TextChange;
-
-namespace Microsoft.VisualStudio.LanguageServerClient.Razor.Extensions
-{
-    internal static class RoslynTextChangeExtensions
```

Comment:
Renamed class to `TextChangeExtensions` to better fit the rest of the extension classes. (Ditto with some other classes below)